examples/kiva/dash.py

```diff
@@ -34,7 +34,8 @@ def dash(sz=(1000, 1000)):
     gc.close_path()
     gc.draw_path()
     t2 = perf_counter()
-    file_path = tempfile.mktemp(suffix='.bmp')
+    with tempfile.NamedTemporaryFile(suffix='.bmp', delete=False) as fid:
+        file_path = fid.name
```

Comment:
Same here. There's no reason to use a context manager if you're not treating the file as something which is actually temporary.